packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -48,13 +49,23 @@ export default connectStore(() => {
 
   return {
     lastShutdownTime: createGetVmLastShutdownTime(),
+    tasks: createGetObjectsOfType('task')
+      .pick(
+        createSelector(
+          (_, { vm }) => vm.current_operations,
+          createCollectionWrapper(operations => Object.keys(operations))
+        )
+      )
+      .filter([{ status: 'pending' }])
```

Comment:
The wrapping array is unnecessary, it's only useful to prevent functions from being interpreted as selectors.

```suggestion
      .filter({ status: 'pending' })
```